checker/run

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -ex
+
+cfg_dir=$(dirname "$0")
+
+. "$cfg_dir"/cfg.sh
```

Comment:
The intention was to have all the checker files under the same directory, being the runner common to all the branches and the configuration specific to each of them. The `options` file at the root of the tree feels better in case we add more configurable parameters later, changed.